README.asciidoc

```diff
@@ -1,59 +1,108 @@
-
+Rudder is an established project with several 10000s of node managed, in companies
+from small to biggest-on-their-field. Typical deployement manages 100 to 1000 nodes.
+The biggest known deployment in 2016 is 7000 nodes.
 
 TIP: Rudder is open source – we welcome ideas, comments, criticism, patches and any other contributions! So let's get a chat we our community https://www.rudder-project.org/site/support/chat-mailing-lists/[on irc]!
 
 
+image::readme-resources/dashboard.png[Rudder Dashboard]
+
+
+
+
+
+
 === Feature overview
 
-* Automate common system administration tasks (installation, configuration)
-* Enforce configuration over time (configuring once is good, ensuring that configuration is valid and automatically fixing it is better)
+* Web interface to manage nodes and define policies
+  ** https://www.rudder-project.org/site/documentation/screenshots/
+* Continuously assesses drift compared to defined policies (with or without auto-healing)
+* Continuously enforces configuration over time (configuring once is good,
+  ensuring configuration is always valid and automatically fixing it is better)
 * Inventory of all managed nodes
-* Web interface to configure and manage nodes and their configuration
-* Compliance reporting, by configuration and/or by node
+* Compliance reporting, by configuration and/or by node, with possibility to
+  zoom on non-compliance to understand the problem.
+* Automate common system administration tasks (installation, configuration)
 * Full API to interact with Rudder server
 
-Screenshots:: https://www.rudder-project.org/site/documentation/screenshots/
+
+[horizontal]
+Rudder site:: http://rudder-project.org
+User documentation:: http://www.rudder-project.org/doc-latest/
+API documentation:: http://www.rudder-project.org/rudder-api-doc/
+Mailing list, irc:: https://www.rudder-project.org/site/community/mailing-lists/
+Bug report:: http://www.rudder-project.org/redmine/projects/rudder/issues
+Source:: https://github.com/Normation/rudder
+
 
 === Testing Rudder
 
 ===== Demo site
 
-You can see a demo version of Rudder:
-
-* Long term support version: http://demo-stable.rudder-project.org/
-* Latest release: http://demo-latest.rudder-project.org/
+You can see a demo version of Rudder: http://demo.rudder-project.org/
 
 ===== Linux packages
 
-The easiest way to test and use Rudder is to install it from the provided Linux packages. 
+The easiest way to test and use Rudder is to install it from the provided Linux packages.
 For the server, the main .rpm-based (RedHat, CentOS, SLES) and .deb-based (Debian, Ubuntu)
 distribution are supported. See https://www.rudder-project.org/site/get-rudder/downloads/
-for details about repository and available versions. 
+for details about repository and available versions.
 
 ===== Vagrant
```

Comment:
We may move this just after the demo, and add a comment to explain the differences with the demo (real agent, etc.)